src/infra/HttpServiceImpl.ts

```diff
@@ -58,7 +58,7 @@ export class HttpServiceImpl implements HttpService {
     reqInit: RequestInit
   ): Promise<Response> {
     const urlWithParams = addParamsToURL(url, queryParams);
-    if (window.fetch) {
+    if (typeof(window) !== 'undefined' && window.fetch) {
```

Comment:
would globalThis.fetch work instead of window.fetch?

Reply:
globalThis doesn't have IE support. We could polyfill it, but that seems a little overkill. Thoughts?

Reply:
ah ok was just wondering if it would be slightly nicer